l3kernel/l3seq.dtx

```diff
@@ -1411,6 +1450,64 @@
 % \end{macro}
 % \end{macro}
 %
+% \begin{macro}
+%   {
+%     \seq_set_regex_extract:Nnn, \seq_set_regex_extract:cnn,
+%     \seq_gset_regex_extract:Nnn, \seq_gset_regex_extract:cnn
+%   }
+% \begin{macro}
+%   {
+%     \seq_set_regex_extract:NNn, \seq_set_regex_extract:cNn,
+%     \seq_gset_regex_extract:NNn, \seq_gset_regex_extract:cNn
+%   }
+% \begin{macro}
+%   {
+%     \seq_set_regex_split:Nnn, \seq_set_regex_split:cnn,
+%     \seq_gset_regex_split:Nnn, \seq_gset_regex_split:cnn
+%   }
+% \begin{macro}
+%   {
+%     \seq_set_regex_split:NNn, \seq_set_regex_split:cNn,
+%     \seq_gset_regex_split:NNn, \seq_gset_regex_split:cNn
+%   }
+%    \begin{macrocode}
+\cs_new_protected:Npn \seq_set_regex_extract:Nnn #1#2#3
+  { \regex_extract:nnN {#3} {#2} #1 }
```

Comment:
`\regex_extract:(n|N)nN` are undefined, there are only `\regex_replace_(once|all):nnN` and friends. (Add some tests?)

BTW what's originally suggested in #890 was
```tex
\cs_new_protected:Npn \seq_set_regex_extract:Nnn #1#2#3
  { \regex_extract_all:nnN {#3} {#2} #1 }
```

Reply:
> `\regex_extract:(n|N)nN` are undefined, there are only `\regex_replace_(once|all):nnN` and friends. (Add some tests?)

I'd thought about tests, but as these are thin wrappers hope that once we have them right, there really should be no need to recheck ...

Reply:
> BTW what's originally suggested in #890 was
> 
> ```
> \cs_new_protected:Npn \seq_set_regex_extract:Nnn #1#2#3
>   { \regex_extract_all:nnN {#3} {#2} #1 }
> ```

I should really make it more clear: do we need all two sets of wrappers `\seq_set_regex_extract_(one|all):...` or we only wrap `\regex_extract_all:...`? @blefloch 

If both groups of `\regex_replace_(one|all):...` can have `\tl_regex_replace_(one|all):...` wrappers, then why not `\seq_set_regex_extract_(one|all):...`?

Reply:
I'd wondered that - my take was that as this is a wrapper for convenience, we might just go for the common case